addplayer.css

```diff
@@ -0,0 +1,55 @@
+* {
+    box-sizing: border-box;
```

Comment:
For styles that are repeated on every page like this one - have a main css file that you link on every page and then add styles that are specific to each page in their own style sheets. Reduce duplicated code wherever you can 😄